cdc/model/changefeed.go

```diff
@@ -31,6 +33,50 @@ type ChangeFeedDetail struct {
 	// The ChangeFeed will exits until sync to timestamp TargetTs
 	TargetTs uint64          `json:"target-ts"`
 	Info     *ChangeFeedInfo `json:"-"`
+
+	filter              *filter.Filter
+	FilterCaseSensitive bool          `json:"filter-case-sensitive"`
+	FilterRules         *filter.Rules `json:"filter-rules"`
```

Comment:
Is `black-white-list` better name here, which has been used in DM and lightning.

Reply:
Besides,  how about adding a user-friendly way to add `black-white-list` config when they create a new changefeed. (Maybe add it in `cdc cli`, just for QA test temporarily)

Reply:
1. The usage of `bwlist` seems confusing in DM, sometimes it references the filter itself and sometimes the rules, so I chose to reuse the name exposed by the filter package.
1. I can add that in a separate PR.